setup.py

```diff
@@ -49,7 +52,7 @@
 
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.6, 2.7',
```

Comment:
This isn't a valid [trove classifier](https://pypi.python.org/pypi?%3Aaction=list_classifiers). We need to add another classifier below with 2.7 in it instead of adding to the existing one.